src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -68,7 +50,9 @@ pub struct KvmVcpu {
     pub mmio_bus: Option<devices::Bus>,
 
     mpidr: u64,
+    template_registers_ids: Vec<u64>,
```

Comment:
nit: may template_register_ids sounds better?

Reply:
My logic is:
registers_ids -> there are many registers and each has an id
register_ids -> there is one register and it has many ids
...naming is hard